client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -537,6 +537,11 @@ const CheckoutSummaryBody = styled.div`
 		max-width: 328px;
 		position: fixed;
 		width: 100%;
+		overflow-y: auto;
+		height: 100vh;
+	}
+	.card {
```

Comment:
I'm not 100% sure about this, but I think that maybe, even though this selector is inside the `styled` call, it might apply globally. I think you should scope it to just this component using the `&` shortcut to say "only cards inside this element".

```suggestion
	& .card {
```